testplan/common/remote/remote_resource.py

```diff
@@ -70,12 +77,21 @@ def get_options(cls):
             ConfigOption("workspace", default=pwd()): str,
             ConfigOption("workspace_exclude", default=[]): Or(list, None),
             ConfigOption("remote_runpath", default=None): str,
+            # no-op now, will be removed soon
```

Comment:
shall we just remove, don't know when will be the next time we revisit this part